test-app/tests/integration/components/input-field-test.gts

```diff
@@ -196,4 +201,31 @@ module('Integration | Component | Fields | Input', function (hooks) {
     </template>);
     assert.dom('[data-root-field="selector"]').exists();
   });
+
+  test('it renders a `<:secondary>` block that tracks the input value length', async function (assert) {
+    class Context {
+      @tracked count = 0;
+
+      @action
+      handleChange(value: string) {
+        this.count = value.length;
+      }
+    }
+
+    let ctx = new Context();
```

Comment:
Do we need this actually for this test? Seems this is never read from?

Reply:
I think it's used over on line 218 in the @onChange: https://github.com/CrowdStrike/ember-toucan-core/pull/115/files#diff-b1d74a6d42d39cd9e5f8eb62146064c2694715629721f598278442b2a9249855R218

Reply:
For some more context, I update the `current` for the consumer (user of the component I mean) over here: https://github.com/CrowdStrike/ember-toucan-core/pull/115/files#diff-1f29edf2e3cdd8fbd4bbbdf9d06e2a675cbc42ffab5bc1a77b371214c86a5b46R70

Reply:
I'm wondering if we even need this Context object as well. It looks like the test has everything it needs without this, as you're only asserting the character count works as you type into it, no?